### ISSUE TITLE
PackageNotFoundError: Package missing in current win-64 channels: - python 3.7*

### ISSUE THREAD

**Issue by BenAbabaei** [tensorflow/tensorflow#27610]
**System information**
- Windows 10
- TensorFlow installed from GitHub (devtools::install_github("rstudio/tensorflow"))
- TensorFlow version: latest per 2019.04.08 (r1.13)
- Python version: 3.7
- Bazle version: 0.24.1 (bazel-0.24.1-windows-x86_64.exe)
- Installed using 'default' for windows, i.e. 'system'. The same with 'conda'

**I followed issue [20517](https://github.com/tensorflow/tensorflow/issues/20517), but I keep getting this exception in RStudio with R/3.5.0:**

```
> install_tensorflow()
Creating r-tensorflow conda environment for TensorFlow installation...
Fetching package metadata ...........

PackageNotFoundError: Package missing in current win-64 channels: 
  - python 3.7*

Error: Error 1 occurred creating conda environment r-tensorflow
```
**I get the same error when I try installing Kras:**

```
library(keras)
install_keras()

Creating r-tensorflow conda environment for TensorFlow installation...
Fetching package metadata ...........

PackageNotFoundError: Package missing in current win-64 channels: 
  - python 3.7*

Error: Error 1 occurred creating conda environment r-tensorflow

```

**I tried to install version 1.13.1, but after installation, I got this error when running 'sess = tf$Session()':**

```
Error: Installing TensorFlow requires a 64-bit version of Python 3.5 or 3.6

Please install 64-bit Python 3.5 or 3.6 to continue, supported versions include:

 - Anaconda Python (Recommended): https://www.anaconda.com/download/#windows
 - Python Software Foundation   : https://www.python.org/downloads/
```

**Comment by ymodak** [comment]
After installing TF you need to specify the ```TENSORFLOW_PYTHON``` environment variable(https://tensorflow.rstudio.com/tensorflow/articles/installation.html#locating-tensorflow). Can you please confirm? Thanks!

**Comment by BenAbabaei** [comment]
I reinstalled everything and apparently there was a problem with a new warning in the new version of conda, which is now fixed. So, my problem has been fixed. Thanks for the response.

**Comment by star1327p** [comment]
I also encountered the `PackageNotFoundError` while installing TensorFlow. I took the following steps and eventually resolved the issue:

1. Find the PC path to R: `C:\Program Files\R\R-3.6.0\bin`

2. From the Start menu, open the Anaconda Prompt (black window terminal) and change to the directory above. The command is `cd C:\Program Files\R\R-3.6.0\bin`.

3. In the Anaconda Prompt, run `R` or `.\R.exe` to get into the R command line.

4. In the R command line, include `library(tensorflow)` and run `install_tensorflow()`.

5. Check `tf_version()` to be `1.13`.

To install keras, just change the commands in Step 4 to `library(keras)` and `install_keras()`.
